R/simulate.R

```diff
@@ -26,9 +26,10 @@ simulate_uniform_cases <- function(sample_size = 1000, t = 60) {
 #' If the rate parameter is 0, then this function defaults to the uniform
 #' distribution.
 #'
-#' @param r The rate parameter for the exponential distribution. Defaults to 0.2.
+#' @param r The rate parameter for the exponential distribution. Defaults to
```

Comment:
Is it clearer/more accurate to describe this as an exponential growth rate and avoid saying exponential distribution?

Reply:
Maybe, though perhaps out of scope for this issue? (I've not changed this text aside from the spacing.)

Reply:
I agree w/ promoting to a new issue